services/driver-service/src/test/java/com/Harevich/driverservice/e2e/CarStepDefinitions.java

```diff
@@ -0,0 +1,129 @@
+package com.Harevich.driverservice.e2e;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+import io.cucumber.java.en.Then;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.path.json.JsonPath;
+import org.junit.Assert;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.UUID;
+
+import static com.Harevich.driverservice.constants.TestConstants.BASIC_SERVER_HOST;
+import static com.Harevich.driverservice.constants.TestConstants.RELATIVE_CREATE_CAR_URL;
+import static com.Harevich.driverservice.constants.TestConstants.RELATIVE_CREATE_DRIVER_URL;
+import static com.Harevich.driverservice.constants.TestConstants.RELATIVE_DELETE_CAR_URL;
+import static com.Harevich.driverservice.constants.TestConstants.RELATIVE_GET_ID_CAR_URL;
+import static com.Harevich.driverservice.constants.TestConstants.RELATIVE_UPDATE_CAR_URL;
+
+
+@Transactional
+public class CarStepDefinitions {
+
+    private Response response;
+    private String requestBody;
+    private static UUID carId;
+
+    @Given("There is car service request")
+    public void thereIsCarServiceRequest(String body) {
+        this.requestBody = body;
+    }
+
+    @When("creates the car")
+    public void createsTheCar() {
+        response = RestAssured.given()
+                .contentType("application/json")
+                .body(requestBody)
+                .post(BASIC_SERVER_HOST + RELATIVE_CREATE_CAR_URL);
+
+        if (response.getStatusCode() == 201) {
+            carId = UUID.fromString(response.jsonPath().getString("id"));
+        }
+    }
```

Comment:
Здесь и далее:
Строковые литералы и magic numbers следует выносить в константы, либо использовать уже созданные константы.
```suggestion
    @When("creates the car")
    public void createsTheCar() {
        response = RestAssured.given()
                .contentType(MediaType.APPLICATION_JSON_VALUE)
                .body(requestBody)
                .post(BASIC_SERVER_HOST + RELATIVE_CREATE_CAR_URL);

        if (response.getStatusCode() == HttpStatus.CREATED.value()) {
            carId = UUID.fromString(response.jsonPath().getString("id"));
        }
    }
```